packages/loopring_v3/contracts/aux/migrate/Drainer.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+
+import "../../lib/ReentrancyGuard.sol";
+import "../../lib/Claimable.sol";
+import "../../lib/Drainable.sol";
+
```

Comment:
This contract enables an alternative approach of getting back assets from Loopring Exchange v1. Now you don't have to withdraw using merkle proofs (very expensive); instead, all assets will be distributed on Loopring Exchange v2 - Loopring's new zkRollup impelmentation. Please activate and unlock your address on https://exchange.loopring.io to claim your assets.